app/javascript/mastodon/features/list_timeline/index.js

```diff
@@ -156,6 +166,24 @@ class ListTimeline extends React.PureComponent {
             <button className='text-btn column-header__setting-btn' tabIndex='0' onClick={this.handleDeleteClick}>
               <i className='fa fa-trash' /> <FormattedMessage id='lists.delete' defaultMessage='Delete list' />
             </button>
+
+            { replies_policy !== undefined && (
+              <div>
+                <div className='column-settings__row'>
+                  <fieldset>
+                    <legend><FormattedMessage id='lists.replies_policy.title' defaultMessage='Show replies to:' /></legend>
+                    { ['no_replies', 'list_replies', 'all_replies'].map(policy => (
+                      <div className='setting-radio'>
+                        <input className='setting-radio__input' id={['setting', 'radio', id, policy].join('-')} type='radio' value={policy} checked={replies_policy === policy} onChange={this.handleRepliesPolicyChange} />
```

Comment:
Plain radio buttons look rather unsightly in there, can we use the toggles we use in other column settings?

Reply:
Someone pointed out that we already have radio buttons somewhere else in the interface (directory settings), I'll have a look to reuse that styling.